_inc/client/performance/search.jsx

```diff
@@ -14,59 +14,95 @@ import { withModuleSettingsFormHelpers } from 'components/module-settings/with-m
-						) }{' '}
-					</p>
-					{ plan_is_business && (
+function toggleModuleFactory( { getOptionValue, isSearchPlan, toggleModuleNow, updateOptions } ) {
+	return module => {
+		toggleModuleNow( module );
+		if ( isSearchPlan && getOptionValue( 'search' ) ) {
+			updateOptions( { instant_search_enabled: true } );
+		}
+	};
+}
+
+function toggleInstantSearchFactory( { isSearchPlan, getOptionValue, updateOptions } ) {
+	return () => {
+		if ( isSearchPlan && getOptionValue( 'search' ) ) {
+			updateOptions( {
+				instant_search_enabled: ! getOptionValue( 'instant_search_enabled', 'search' ),
+			} );
+		}
+	};
+}
+
+function Search( props ) {
+	const isModuleEnabled = props.getOptionValue( 'search' );
+	const isInstantSearchEnabled = props.getOptionValue( 'instant_search_enabled', 'search' );
+
+	const toggleModule = useMemo( () => toggleModuleFactory( props ), [
+		props.getOptionValue,
+		props.isSearchPlan,
+		props.toggleModuleNow,
+		props.updateOptions,
+	] );
+	const toggleInstantSearch = useMemo( () => toggleInstantSearchFactory( props ), [
+		props.getOptionValue,
+		props.isSearchPlan,
+		props.updateOptions,
+	] );
+
+	return (
+		<SettingsCard { ...props } module="search" feature={ FEATURE_SEARCH_JETPACK } hideButton>
+			<SettingsGroup
+				hasChild
+				module={ { module: 'search' } }
+				support={ {
+					text: __( 'Jetpack Search supports many customizations.' ),
+					link: 'https://jetpack.com/support/search',
+				} }
+			>
+				<p>
+					{ __(
+						'Help visitors quickly find answers with highly relevant instant search results and powerful filtering hosted in the WordPress.com cloud.'
+					) }{ ' ' }
+				</p>
+				{ ( props.isBusinessPlan || props.isSearchPlan ) && (
+					<Fragment>
 						<ModuleToggle
-							slug="search"
+							activated={ isModuleEnabled }
 							compact
-							activated={ module_enabled }
-							toggling={ this.props.isSavingAnyOption( 'search' ) }
-							toggleModule={ this.props.toggleModuleNow }
+							slug="search"
+							toggleModule={ toggleModule }
+							toggling={ props.isSavingAnyOption( 'search' ) }
 						>
-							{ __(
-								'Replace WordPress built-in search with Jetpack Search, an advanced search experience'
-							) }
+							{ __( 'Enable Jetpack Search' ) }
 						</ModuleToggle>
-					) }
-					{ plan_is_business && module_enabled && (
 						<FormFieldset>
-							<p className="jp-form-setting-explanation">
+							<CompactFormToggle
+								checked={ isInstantSearchEnabled }
+								disabled={ ! props.isSearchPlan || ! isModuleEnabled }
+								onChange={ toggleInstantSearch }
+								toggling={ props.isSavingAnyOption( 'instant_search_enabled' ) }
+							>
+								<span className="jp-form-toggle-explanation">
+									{ __( 'Enable instant search experience (recommended)' ) }
```

Comment:
Suggesting we remove “experience” from this option to keep it cleaner. Also, using Title Case here to match the description below, though it's not a product name. So maybe either use Title Case here or change the description below?

```suggestion
									{ __( 'Enable Instant Search (recommended)' ) }
```

Reply:
I think we are going for lower case for "instant search" since it is not the name of the product.

Reply:
Sounds good, I'll update my suggestion.